lib/client/file-client/FileClient.js

```diff
@@ -1,18 +1,29 @@
 'use strict';
 
-var config = require('./config'),
+var config = require('../../config'),
   q = require('q');
 
-var client = {};
+var $fh;
 
-client.init = function() {
+/**
+ * A FileClient service that will be responsible for file data management
+ *
+ * @param {object} config
+ * @param {object} fhSdk
+ * @constructor
+ */
+var FileClient = function FileClient(config, fhSdk){
```

Comment:
We do not have config used at the moment but it would be used to customize client actions. 
I have couple other tickets that would require config so decided to setup this as passing config seems to be common pattern for modules.